mesontest.py

```diff
@@ -126,12 +126,12 @@ def decode(stream):
         return stream.decode('iso-8859-1', errors='ignore')
 
 def write_json_log(jsonlogfile, test_name, result):
-    jresult = {'name' : test_name,
-              'stdout' : result.stdo,
-              'result' : result.res,
-              'duration' : result.duration,
-              'returncode' : result.returncode,
-              'command' : result.cmd,
+    jresult = {'name': test_name,
+              'stdout': result.stdo,
+              'result': result.res,
+              'duration': result.duration,
+              'returncode': result.returncode,
+              'command': result.cmd,
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->